api/HTMLTrackElement.json

```diff
@@ -75,6 +75,194 @@
           "deprecated": false
         }
       },
+      "ERROR": {
```

Comment:
These are constants, and it looks like for example constants on `HTMLMediaElement` aren't in BCD. @ddbeck @Elchi3 should we represent constants in BCD? In some ways they're part of the attribute using them, in this case the `readyState` attribute, but they are actual attributes on the interface object and prototype, so unlike enums they're not invisible.

I couldn't find any open issue about this.

Reply:
Turns out it's being discussed in https://github.com/mdn/browser-compat-data/issues/7585, let's talk over there.